src/niscope/metadata/functions_addon.py

```diff
@@ -425,29 +495,29 @@
 
 # Override the 'python' name for some functions.
 functions_python_name = {
-    'FetchDispatcher':            { 'python_name': 'fetch',                           },
-    'FetchDouble':                { 'python_name': 'fetch',                           },
+    'FetchDispatcher':            { 'python_name': 'fetch', },
+    'FancyFetch':                 { 'python_name': 'fetch', },
 }
 
 functions_method_templates = {
-    'FetchBinary8':                                  { 'method_templates': [
-        { 'session_filename': 'numpy_read_method', 'method_python_name_suffix': '_into', },
+    'FetchBinary8':    { 'method_templates': [
+        { 'session_filename': 'numpy_read_method', 'method_python_name_suffix': '_into_numpy', },
     ], },
-    'FetchBinary16':                                 { 'method_templates': [
-        { 'session_filename': 'numpy_read_method', 'method_python_name_suffix': '_into', },
+    'FetchBinary16':   { 'method_templates': [
+        { 'session_filename': 'numpy_read_method', 'method_python_name_suffix': '_into_numpy', },
     ], },
-    'FetchBinary32':                                 { 'method_templates': [
-        { 'session_filename': 'numpy_read_method', 'method_python_name_suffix': '_into', },
+    'FetchBinary32':   { 'method_templates': [
+        { 'session_filename': 'numpy_read_method', 'method_python_name_suffix': '_into_numpy', },
     ], },
-    'Fetch':                { 'method_templates': [
+    'Fetch':           { 'method_templates': [
         { 'session_filename': 'default_method', 'method_python_name_suffix': '', },
-        { 'session_filename': 'numpy_read_method', 'method_python_name_suffix': '_into', },
+        { 'session_filename': 'numpy_read_method', 'method_python_name_suffix': '_into_numpy', },
```

Comment:
The reason these were renamed is because they don't work as is with array.array, right?
And the plan is to fix that and change the name back? If so, I don't think the rename is necessary, just an issue.